plugins/lsp/language-servers/ltex-settings.nix

```
@@ -3,7 +3,7 @@ with lib;
 {
   enabled =
     helpers.defaultNullOpts.mkNullable (with types; either bool (listOf str))
```

```suggestion
    helpers.defaultNullOpts.mkNullable (with helpers.nixvimTypes; either bool (listOf (maybeRaw str)))
```

I think all these options should accept string or raw.

---

I'd be tempted to move `with helpers.nixvimTypes` to the top of the file, as this could get rather verbose otherwise and everything here's option defs anyway.

---

This would also mean dropping "cosmetic" from the commit title

---

Done.